test

```diff
@@ -1,35 +1,28 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
```

Comment:
I'd keep the old version of this line. Also don't need to have -e and then set -e right below.